jetty-embedded-10/src/main/java/org/jboss/arquillian/container/jetty/embedded_10/ArquillianAppProvider.java

```diff
@@ -98,6 +98,7 @@ protected App createApp(final Archive<?> archive) {
         try {
             if (this.config.isUseArchiveNameAsContext()) {
                 Path tmpDirectory = Files.createTempDirectory("arquillian-jetty");
+                tmpDirectory.toFile().deleteOnExit();
```

Comment:
Two problems with this.

A) the deleteOnExit doesn't handle directories really well.
B) the deleteOnExit only works on proper JVM shutdown (a Ctrl+C would bypass this cleanup)

I personally wouldn't use File.deleteOnExit for anything.
If this is a directory passed into Jetty, let Jetty clean it up.

Reply:
works fine for me using Linux (probably osx)